railties/lib/rails/generators/rails/app/templates/github/ci.yml.tt

```diff
@@ -9,6 +9,8 @@ jobs:
 <%- unless skip_brakeman? -%>
   scan_ruby:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
Hum, actually fail fast only applies to matrixes, so I guess it's only really an annoyance for gems. I guess this isn't needed.